nss_simpleauthd/src/group.rs

```diff
@@ -0,0 +1,47 @@
+use authd::types::ToNSS;
+use futures::executor::block_on;
+use libnss::{
+    group::{Group, GroupHooks},
+    interop::Response,
+};
+use tarpc::context;
+
+use crate::{AuthdGroup, RPC};
```

Comment:
define the struct in this module? and then reference from `ffi` (use `pub(crate)` liberally to solve any namespace issues)